src/Scripts/Game/Features/ID/Licence.c

```diff
@@ -0,0 +1,15 @@
+class Licence
+{
+	protected RTTextureWidget m_wRenderTargetTextureWidget;
+	TextWidget licenceTextWidget;
+	
+	void Licence(string widgetname, string displayname, Widget m_wPIPRoot)
```

Comment:
```suggestion
	void Licence(string widgetname, string displayname, Widget wPIPRoot)
```

Reply:
According to the Bohemia coding rules :
_Member variables are prefixed with m_ , eventually one-letter type prefix, and use PascalCase_

Reply:
This is merely a derived value from what the method receives when called it can really be anything you want just in camelCase so if the w is there just to satisfy that then its fine as is. No m_ needed as its not a declared member variable of the class.

Your:  protected ref Widget m_wPIPRoot;   this is fine.
Within this void License method (pppssst its spelled License, no second c its an s) it can just be wPIPRoot as @ThibaudetTevai stated.